src/index.ts

```diff
@@ -56,6 +57,45 @@ passport.use('local', new LocalStrategy(
   }
 ))
 
+if ( process.env.GOOGLE_CLIENT_ID && process.env.GOOGLE_CLIENT_SECRET ) {
+  passport.use('google', new GoogleStrategy({
+      clientID: process.env.GOOGLE_CLIENT_ID,
+      clientSecret: process.env.GOOGLE_CLIENT_SECRET,
+      callbackURL: '/auth/google/redirect',
+    },
+    (issuer, profile, done  ) => {
```

Comment:
This should use async/await and avoid `.then()`s.

Reply:
It also looks like the ESLint formatting rules aren't being applied here. Those are deprecated anyway and I need to switch the project over to prettier so I'm not too worried about that. We can merge this with whatever formatting.

Reply:
Done in https://github.com/Wingysam/Christmas-Community/pull/138/commits/173add9093dd9ad4ab64fbeb9751ead78477d44e.